indica/operators/impurity_concentration.py

```diff
@@ -76,8 +76,8 @@ def __call__(  # type: ignore
         Parameters
         ----------
         element
-            String specifying the element for which the impurity concentration
-            is desired. It should be given in full lower-case, eg. "beryllium"
+            String specifying the symbol of the element for which the impurity
```

Comment:
Should this, and it's usages, be changed to use element symbol instead of name for consistency with other uses?

Reply:
I think I've caught all of the uses of element name and changed them to symbol but if I've missed any please let me know. Do you mean should the change from element name to element symbol have been made at all?